SimPEG/Maps.py

```diff
@@ -604,40 +715,40 @@ def _sc2phaseEMTRandSpheroidstransform(self, phi1):
 
         return sige2
 
-    def _sc2phaseEMTRandSpheroidsinversetransform(self, sige):
+    def _sc2phaseEMTSpheroidsinversetransform(self, sige):
 
-        R0 = getR(self.sigma0, sige, self.alp0)
-        R1 = getR(self.sigma1, sige, self.alp1)
+        R0 = getR(self.sigma0, sige, self.alpha0, self.orientation0)
+        R1 = getR(self.sigma1, sige, self.alpha1, self.orientation1)
 
         num = -(sigma0 - sige)*R0
         den = (sigma1-sige)*R1 - (sigma0-sige)*R0
 
         return num/den
 
-    def _sc2phaseEMTRandSpheroidstransformDeriv(self, sige, phi1):
+    def _sc2phaseEMTSpheroidstransformDeriv(self, sige, phi1):
```

Comment:
So, I guess inversion parameter is phi1 (volume fraction) ?
I am not sure why phi is not set as a property... any reason why?

Reply:
here, phi is the inversion model, so it should be an input, rather than a property. I have added docs to hopefully clarify